src/pybind/mgr/snap_schedule/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+
+import os
+
+if 'CEPH_SNAP_SCHEDULE_UNITTEST' not in os.environ:
+    from .module import Module
```

Comment:
Would be great to take https://github.com/ceph/ceph/blob/master/src/pybind/mgr/insights/__init__.py as an example.